tests/test_labelled_pandas.py

```diff
@@ -450,3 +450,12 @@ def test_reset_index():
     assert not np.array_equal(new.index, ldf.reset_index())
     new.reset_index(inplace=True)
     assert_array_equal(new.index, ldf.reset_index().index)
+
+
+def test_drop_labels():
+    ldf = test_LabelledDataFrame_index()
+    assert ldf.islabelled()
+    nolabels = ldf.drop_labels()
+    assert not nolabels.islabelled()
+    assert_array_equal(nolabels.drop_labels(), nolabels)
```

Comment:
Why this line? Isn't that the same by construction?

For addressing the problem in this PR's description, wouldn't it be better to have test_LabelledDataFrame_index() return both lframe and frame, and then do testing with both?

Reply:
It is, but before `nolabels.drop_labels()` returned it could return `None`. Update to `assert_frame_equal` which also tests indices.